spec.bs

```diff
@@ -185,6 +185,20 @@ dictionary AppHistoryReloadOptions : AppHistoryNavigationOptions {
 };
 </xmp>
 
+<div algorithm>
+  An {{AppHistory}} |appHistory| <dfn for="AppHistory">has entries and events disabled</dfn> if the following steps return true:
+
+  1. Let |browsingContext| be |appHistory|'s [=relevant global object=]'s [=Window/browsing context=].
+
+  1. If |browsingContext| is null, then return true.
+
+  1. If |browsingContext| is <a spec="HTML">still on its initial `about:blank` `Document`</a>, then return true.
```

Comment:
This is a bit surprising. It essentially inherits a bunch of state from the parent in this case. Is there something missing that would make supporting history problematic?

Reply:
I think it's doable, but so far I've been saying that the initial about:blank is a scary place (e.g., most navigations from it are auto-converted into replacements; you can't navigate back to it currently; etc.) so it'd be best to avoid exposing that complexity.

Reply:
It's easier to allow later than to disallow later so I'm okay with it, FWIW.